dev/TabView/TabViewAutomationPeer.cpp

```diff
@@ -18,6 +18,10 @@ TabViewAutomationPeer::TabViewAutomationPeer(winrt::TabView const& owner)
 // IAutomationPeerOverrides
 winrt::IInspectable TabViewAutomationPeer::GetPatternCore(winrt::PatternInterface const& patternInterface)
```

Comment:
Yes, we will also need to update TabViewItemAutomationPeer to support SelectionItem. It currently derives from ListViewItem which will point to the ListView as the selection container which is not what we want here. I think we can do what we do in NavigationViewItem [here](https://github.com/microsoft/microsoft-ui-xaml/blob/master/dev/NavigationView/NavigationViewItemAutomationPeer.cpp).